src/main/java/hudson/tasks/Mailer.java

```diff
@@ -610,5 +652,6 @@ public UserProperty newInstance(StaplerRequest req, JSONObject formData) throws
     /**
      * Debug probe point to be activated by the scripting console.
      */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "It's used for debugging")
```

Comment:
the script console can access private fields...  Groovy has no concept of private!